strawberry/types/fields/resolver.py

```diff
@@ -38,6 +42,10 @@ def __init__(
     def __call__(self, *args, **kwargs) -> T:
         return self.wrapped_func(*args, **kwargs)
 
+    @classmethod
+    def default_resolver(cls):
```

Comment:
```suggestion
    def default_resolver(cls) -> StrawberryResolver:
```

Reply:
Is mypy not clever enough to figure that out since I’m returning `cls`?

Reply:
Maybe it is, but I tend towards being explicit